multiqc/modules/snpeff/snpeff.py

```diff
@@ -49,22 +52,65 @@ def __init__(self):
         # Report sections
         self.add_section (
             name = 'Variants by Genomic Region',
+            description = (
+                "The stacked bar plot shows locations of detected variants in"
+                " the genome and the number of variants for each location."
+            ),
+            helptext = (
+                "The upstream and downstream interval size to detect these "
+                "genomic regions is 5000bp by default."
+            ),
             anchor = 'snpeff-genomic-regions',
             plot = self.count_genomic_region_plot()
         )
         self.add_section (
             name = 'Variant Effects by Impact',
+            description = (
+                "The stacked bar plot shows the putative impact of detected "
+                "variants and the number of variants for each impact."
+            ),
+            helptext = (
+                "There are four levels of impacts predicted by SnpEff:"
+                "<ul>"
```

Comment:
Nice! You can add lists as markdown instead if you like:

```
There are four levels of impacts predicted by SnpEff:

* **High**: High impact (like stop codon)
* **Moderate**: Middle impact (like same type of amino
```
etc.. Should render as:

---

There are four levels of impacts predicted by SnpEff:

* **High**: High impact (like stop codon)
* **Moderate**: Middle impact (like same type of amino